tests/unit/models/ObjectMD.js

```diff
@@ -257,6 +257,23 @@ describe('ObjectMD class setters/getters', () => {
         md.clearMetadataValues();
         assert.strictEqual(md.getUserMetadata(), undefined);
     });
+
+    it('ObjectMD::microVersionId', () => {
+        assert.strictEqual(md.getMicroVersionId(), null);
```

Comment:
This line looks like a separate test, that it defaults to none and that you can update it.